practitioner-service/src/main/java/uk/ac/ox/ndph/mts/practitioner_service/repository/FhirContextWrapper.java

```diff
@@ -46,12 +46,12 @@ public String prettyPrint(IBaseResource resource) {
      * @param input the Bundle to save
      * @return the returned Bundle object from FHIR endpoint
      */
-    public Bundle executeTransaction(String uri, Bundle input) throws FhirServerResponseException {
+    public Bundle executeTransaction(Bundle input) throws FhirServerResponseException {
         try {
-            return fhirContext.newRestfulGenericClient(uri).transaction()
+            return fhirContext.newRestfulGenericClient(fhirUri).transaction()
                     .withBundle(input).execute();
         } catch (BaseServerResponseException ex) {
-            throw new FhirServerResponseException("Problem executing transaction with bundle at: " + uri, ex);
+            throw new FhirServerResponseException("Problem executing transaction with bundle at: " + fhirUri, ex);
```

Comment:
Can you add this exception text to the enum class in the same package and use is from there?

Reply:
Fixed.